pkg/apputils/utils.go

```diff
@@ -160,7 +164,7 @@ func getGitClients(ctx context.Context, url, configUrl, namespace string, isHelm
 	}
 
 	if isExternalConfig {
-		configRepoClient, configRepoErr := authsvc.CreateGitClient(ctx, targetName, namespace, utils.SanitizeRepoUrl(configUrl))
+		configRepoClient, configRepoErr := authsvc.CreateGitClient(ctx, targetName, namespace, configUrl)
```

Comment:
Are we sure this doesn't break anything?  If you follow `configUrl` as it is being passed down from different commands it can vary wildly.  I would double check to make sure this is the right change.  

💡 I wonder if the url should even be passed around, it might be worth a second pass refactoring to have most places take a `NormalizedUrl` instead of a `string`.

Reply:
This one has been normalized but i think you are correct. If everything passes a normalized URL you never have to wonder.

Reply:
this task needs to be handled in a different PR so we can do it right.